src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBidAction.tsx

```diff
@@ -12,19 +12,32 @@ export interface ArtworkSidebarBidActionProps {
 }
 
 export interface ArtworkSidebarBidActionState {
-  nextMaxBidCents?: number
+  selectedMaxBidCents?: number
 }
 
 export class ArtworkSidebarBidAction extends React.Component<
   ArtworkSidebarBidActionProps,
   ArtworkSidebarBidActionState
 > {
   state: ArtworkSidebarBidActionState = {
-    nextMaxBidCents: null,
+    selectedMaxBidCents: null,
   }
 
   setMaxBid = (newVal: number) => {
-    this.setState({ nextMaxBidCents: newVal })
+    this.setState({ selectedMaxBidCents: newVal })
+  }
+
+  redirectToRegister = () => {
+    const { sale } = this.props.artwork
+    window.location.href = `${sd.APP_URL}/auction-registration/${sale.id}`
+  }
+
+  redirectToBid = (firstIncrement: number) => {
+    const { id, sale } = this.props.artwork
+    const bid = this.state.selectedMaxBidCents || firstIncrement
+    window.location.href = `${sd.APP_URL}/auction/${
```

Comment:
I hope we do not need to worry about user presence here and the bid url will take care of that.